courses/util.py

```diff
@@ -133,6 +134,15 @@ def add_college_requirements(course, college_reqs):
         req.courses.add(course)
 
 
+def relocate_reqs_from_rests(rests, reqs):
+    if any(r.requirement_description == 'Humanities & Social Science Sector' for r in rests):
```

Comment:
It might be possible to generalize this a bit more, with a list and an outer for loop. In case this comes up that there's another restriction that we want to treat as a requirement?